Realm/RLMAccessor.mm

```diff
@@ -56,11 +58,24 @@ typedef NS_ENUM(char, RLMAccessorCode) {
     RLMAccessorCodeBoolObject,
 };
 
-// long getter/setter
-static inline long long RLMGetLong(__unsafe_unretained RLMObjectBase *const obj, NSUInteger colIndex) {
+template<typename T>
+static T get(__unsafe_unretained RLMObjectBase *const obj, NSUInteger index) {
```

Comment:
What do you mean by that? `RLMGetLong` isn't visible to users of our API as far as I understand.